content/en/code-of-conduct.md

```diff
@@ -80,4 +80,4 @@ The Committee will respond to any report as soon as possible, and at most within
 
 We are thankful to the groups behind the following documents, from which we drew content and inspiration:
 
-- [The SciPy Code of Conduct](https://docs.scipy.org/doc/scipy/reference/dev/conduct/code_of_conduct.html)
+- [The SciPy Code of Conduct](https://scipy.github.io/devdocs/dev/conduct/code_of_conduct.html)
```

Comment:
The github.io site doesn't seem like the best solution to me?  That seems more of a deployment detail.  Maybe rather: https://docs.scipy.org/doc/scipy/dev/conduct/code_of_conduct.html maybe the "reference" part of the URL went away at some point.

Reply:
🤷‍♂️ 
I used the link that SciPy uses on it's website, figuring that was the authoritative source.
Happy to change to this version though, it seems to work.

Reply:
It looks like https://scipy.org/community/ and the footer on that site use https://docs.scipy.org/doc/scipy/dev/conduct/code_of_conduct.html consistently. So +1 for using that link.

Reply:
OK, I will go with that, but let the record reflect that SciPy started using that URL today:
https://github.com/scipy/scipy/pull/16963/files
😉

Reply:
Ah, I did not realize that.